src/datatrove/data.py

```diff
@@ -39,14 +45,14 @@ class Document:
             a unique id (string) for this sample
         media: list[Media]
             The media associated with the document
-        metadata: dict[str, Any]
+        metadata: dict[str, str | int | float | bool]
             a dictionary where any additional info may be stored
     """
 
     text: str
     id: str
     media: list[Media] = field(default_factory=list)
-    metadata: dict[str, Any] = field(default_factory=dict)
+    metadata: dict[str, str | int | float | bool] = field(default_factory=dict)
```

Comment:
it can have more than this, such as lists of other things or other dictionaries

Reply:
Changed it to Any.